src/main/java/com/cburch/logisim/std/memory/Ram.java

```diff
@@ -176,10 +176,6 @@ public Bounds getOffsetBounds(AttributeSet attrs) {
     return RamAppearance.getBounds(attrs);
   }
 
-  public MemContents getContents(InstanceState ramState) {
-    return getState(ramState).getContents();
-  }
```

Comment:
Is this member function never used?

Reply:
I moved it to the Mem class since it needs to exist for both RAM and ROM objects with the new code to load RAM and ROM.